config/when/when.go

```diff
@@ -165,6 +167,25 @@ func (w *When) validateExists() error {
 	return newCondFailErrorf("no required file existed: %s", w.Exists)
 }
 
+func (w *When) validateNotExists() error {
+	if len(w.NotExists) == 0 {
+		return newUnspecifiedError("not-exists")
+	}
+
+	for _, f := range w.NotExists {
+		if _, err := os.Stat(f); err != nil {
+			if os.IsNotExist(err) {
+				continue
+			}
+			return err
+		}
+
+		return newCondFailErrorf("file exists, but shouldn't: %s", w.NotExists)
```

Comment:
This can then be dropped, since finding a single file should not fail the clause.